dask/dataframe/tests/test_dataframe.py

```diff
@@ -2640,16 +2640,17 @@ def test_to_dask_array_unknown(as_frame):
     assert all(np.isnan(x) for x in result)
 
 
-@pytest.mark.parametrize("lengths", [[2, 3], True])
-@pytest.mark.parametrize("as_frame", [False, False])
-def test_to_dask_array(as_frame, lengths):
-    s = pd.Series([1, 2, 3, 4, 5], name="foo")
+@pytest.mark.parametrize("lengths", [[2, 3], True, True])
+@pytest.mark.parametrize("as_frame", [False, False, False])
+@pytest.mark.parametrize("meta", [None, None, np.array([], dtype="f4")])
+def test_to_dask_array(meta, as_frame, lengths):
```

Comment:
Given that these all have length three I suspect that your intention here is to run three instances of this test.  However, multiple parametrize calls produce cartesian products, so this is actually creating `3**3 == 27` instances of the same test.  Maybe you meant to use 

```python
@pytest.mark.parametrize("lengths,as_frame,meta", [(..., ..., ...), (..., ..., ...), ...])
```

Reply:
good catch. I also notice that as_frame is never True, which seems like it wasn't the original intention of the test. I can fix that as well